openlibrary/templates/books/custom_carousel.html

```diff
@@ -41,7 +41,7 @@
     $ cover_url = False
 
   $if book.get('authors'):
-    $ author_names = [author.name for author in book.authors]
+    $ author_names = [author.name for author in book.authors if author.name]
```

Comment:
```suggestion
    $ author_names = [author.name or _('name missing') for author in book.authors]
```

This might be useful since it can help librarians see that a record needs editing!